DESCRIPTION.md

```diff
@@ -12,6 +12,7 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
   - Added support for `token_file_path` connection parameter to read an OAuth token from a file when connecting to Snowflake.
   - Added support for `debug_arrow_chunk` connection parameter to allow debugging raw arrow data in case of arrow data parsing failure.
   - Added support for `disable_saml_url_check` connection parameter to disable SAML URL check in OKTA authentication.
+  - Added support for the `dataframe_ast` parameter, which allows the caller to propagate dataframe-level AST to Snowflake.
```

Comment:
nit: 3.11.0 is released, this would go to version 3.12.0.

btw is this a public facing feature? if not we probably do not need to call it out, or if it's public facing feature, then it's better to call that out which api this new parameter is added to